src/fortran/fct.cpp

```diff
@@ -354,6 +354,17 @@ extern "C" {
         }
     }
 
+    status_t phase_getpartialmolarintenergies_(const integer* n, doublereal* ie)
```

Comment:
Shouldn't this method be named `th_getpartialmolarintenergies_` for consistency with methods like `th_getentropies_r_`?

Reply:
Agreed. I have modified this function name.